TemplatesVSIX/ReadMe.txt

```diff
@@ -4,5 +4,5 @@ the rebuild will crash because of some properties which are not copatible from t
 	2. Update the source.extension.vsixmanifest file with the needed information (by default, it is pre-populated with information related to Visual Studio version)
 	3. In case the PluginFramework.Build or PluginFramework versions were changed, then it should be also updated inside each template project
 at file SDL Custom Batch Task.vstemplate/SDL Terminology Provider.vstemplate/SDL Trados Studio.vstemplate/SDL Translation Provider.vstemplate,
-so the version from TemplatesVSIX\Packages\Sdl.Core.PluginFramework.1.8.0.nupkg and TemplatesVSIX\Packages\Sdl.Core.PluginFramework.Build.15.1.0.nupkg 
+so the version from TemplatesVSIX\Packages\Sdl.Core.PluginFramework.1.0.0.nupkg and TemplatesVSIX\Packages\Sdl.Core.PluginFramework.Build.16.0.0.nupkg 
```

Comment:
Sdl.Core.PluginFramework version is 2.0.0

Reply:
I've changed the versions.